sa/_db-next/migrations/20171120132231_DropChallengesLockCol.sql

```diff
@@ -0,0 +1,11 @@
+
+-- +goose Up
+-- SQL in section 'Up' is executed when this migration is applied
+
+ALTER TABLE challenges DROP COLUMN LockCol;
```

Comment:
I would leave out this migration. I don't think we are expecting to get the challenges table down to a size where we can migrate it any time soon. I think it's sufficient to leave the LockCol in place but unused.

Reply:
Ok. It's unpleasant leaving unused DB fields laying around but there also doesn't seem to be a chance we can land a migration so I think my hands are tied.